it/ch06.md

```diff
@@ -0,0 +1,63 @@
+# Impostazioni
+
+Il tab `Impostazioni` fornisce strumenti per personalizzare le opzioni di configurazione.
+
+### Portafoglio
+
+![settings-wallet](media/black_settings-wallet.png)
+
+### Nodo Locale
+
+![settings_local-node](media/black_settings-node-local_node.png)
+
+(1) **Avvia/Ferma Nodo Locale:** In base al suo stato corrente, avvia o ferma il nodo.
+(2) **Posizione Blockchain:** Inserimento manuale di un percorso non di default per la blockchain.
+(3) **Flag per il Daemon:** Quando si usa un nodo locale, in questo campo possono essere inserite opzioni aggiuntive a linea di comando.
+(4) **Indirizzo di Bootstrap:** Inserisci l'hostname o l'indirizzo IP del nodo remoto di bootstrap. Vedi la sezione [8.1 nodi di Bootstrap](#8.1-bootstrap-nodes) per una veloce spiegazione su cosa è un nodo di Bootstrap.
+(5) **Porta di Bootstrap:** Inserisci la porta del nodo remoto di bootstrap.
+
+### Nodo Remoto
+
+Usa un nodo remoto, non fa il download della blockchain. Dai uno sguardo alla [sezione 'Nodi Remoti'](#8-about-remote-nodes) di questa guida.
```

Comment:
"non fa il download della blockchain" => "non esegue lo scaricamento della blockchain" or better "non scarica la blockchain in locale".